Classes/Utils/JSONObjects.cs

```diff
@@ -169,7 +175,7 @@ public class CaptureSettings {
         /// </summary>
         public List<FileFormat> fileFormatsCache { get { return _fileFormatCache; } set { _fileFormatCache = value; } }
 
-        private FileFormat _fileFormat = new FileFormat("mp4", "MPEG-4 (.mp4)", true);
+        private FileFormat _fileFormat = new FileFormat("mkv", "Matroska Video (.mkv)", true);
```

Comment:
I don't think we should change this since mkv isn't natively supported by Chrome, Edge, Safari or Firefox. 
Should we change it to fragmented mp4?

Reply:
Yeah, I feel the same!

Reply:
Actually, fragmented mp4 doesn't support replay buffer so maybe mkv makes more sense.

Reply:
I don't think so. It'll switch to regular MP4 automatically when Replay Buffer is on. I disabled the warning if Replay Buffer is active and the output is MP4 since there's not much to lose if the software or PC crashes.

Reply:
I suppose that makes sense, I'm happy with either being used.

Reply:
I think it is fine to keep the default to MP4 (or fragmented) since it is a more commonly supported and that most people are familiar with. Any power user can make the change to MKV when they need it.

Reply:
I'd personally just leave it as is (and not set it to fragmented) and have the user change it if they want.

Reply:
Sounds good, changed it to fragmented.